spring-cloud-stream/src/test/java/org/springframework/cloud/stream/function/SourceToFunctionsSupportTests.java

```diff
@@ -70,7 +71,7 @@
 	@Test
 	public void testFunctionIsAppliedToExistingMessageSource() {
 		try (ConfigurableApplicationContext context = new SpringApplicationBuilder(
-			TestChannelBinderConfiguration.getCompleteConfiguration(FunctionsConfiguration.class)).web(
+			TestChannelBinderConfiguration.getCompleteConfiguration(FunctionsConfiguration.class, TestChannelBinderConfiguration.class)).web(
```

Comment:
Why do you need to add `TestChannelBinderConfiguration` to `TestChannelBinderConfiguration`?

Reply:
This was an oversight.